src/Bicep.Core/Registry/ModuleDispatcherExtensions.cs

```diff
@@ -14,7 +14,7 @@ public static class ModuleDispatcherExtensions
     {
         public static IEnumerable<ArtifactReference> GetValidModuleReferences(this IModuleDispatcher moduleDispatcher, IEnumerable<ArtifactResolutionInfo> artifacts)
```

Comment:
Perhaps doesn't need to be done as part of this PR, but there's still a lot of things with `Module` in the name that are no longer module-specific. Could you perhaps track this as a follow-up item?

Reply:
I agree on the need for renaming but I think that doing this on this PR will be adding confusion. Instead, I would rather address renaming on separate PRs that don't have an effect on behavior.
I can organically address this over time rather than have a specific issue to do this.